app/views/works/show.html.erb

```diff
@@ -6,58 +6,10 @@
 
 <div class="show-page-layout work-show" itemscope itemtype="http://schema.org/CreativeWork" class="row">
 
-
-  <header class="show-title">
-    <% if decorator.genre.present? %>
-      <div class="show-genre">
-        <%= decorator.display_genres %>
-      </div>
-    <% end %>
-
-    <h1>
-      <%= decorator.title %>
-      <%= publication_badge(decorator) %>
-      <% if can?(:read, @work) %>
-        <%= link_to "Edit", admin_work_path(decorator), class: "btn btn-outline-primary" %>
-      <% end %>
-    </h1>
-
-
-    <% if decorator.additional_title.present? %>
-      <ul class="additional-titles list-unstyled">
-        <% decorator.additional_title.each do |title| %>
-          <li><h2><%= title %></h2></li>
-        <% end %>
-      </ul>
-    <% end %>
-
-    <% if decorator.parent.present? || decorator.source.present? %>
-      <ul class="part-of list-unstyled">
-        <% if decorator.parent.present? %>
-          <li>Part of <%= link_to decorator.parent.title, work_path(decorator.parent) %></li>
-        <% end %>
-        <% if decorator.source.present? %>
-          <li> Part of <i><%= decorator.source %></i></li>
-        <% end %>
-      </ul>
-    <% end %>
-
-    <% if decorator.date_of_work.present?  %>
-      <div class="show-date">
-          <ul>
-            <% DateDisplayFormatter.new(decorator.date_of_work).display_dates.each do |interval| %>
-              <li>
-                <span itemprop="date_created">
-                  <%= interval %>
-                </span>
-              </li>
-            <% end %>
-          </ul>
-      </div>
-    <% end %>
-  </header>
+  <%= WorkTitleAndDates.new(@work).display %>
 
   <div class="show-hero">
+    <%= Playlist.new(decorator.audio_members).display if decorator.audio_members.present? %>
```

Comment:
So the downside of only having one template is that your decorator for the non-audio still needs that `audio_members` method. And does that mean the decorator for the non-audio will refuse to include audio in the "ordinary" list of members?

If there IS going to be only one template... there should possibly be only one decorator too, since any method the template calls needs to be available in the decorator, so both decorators need the same methods. (Are they even implemented any different in the two decorators?)

But I'm still kinda liking the idea of two templates -- I think it makes them (and their decorators) both _simpler_, with less conditional logic. And also opens up the idea of choosing to show any Work with either template, instead of it being baked into the code in several places that any work with at least one audio file _always_ displays like our current "oral history" template. With two templates/decorators, it'd just be a question of changing the logic for picking which template/decorator, and you could always show any Work with either one. 

But let's talk about this more in a voice chat?

Reply:
Moved it back to two templates!